tribits/core/package_arch/TribitsSystemDataStructuresMacrosFunctions.rst

```diff
@@ -497,38 +497,38 @@ external packages/TPLs are:
   complient package.  In this case, we just print that we are skipping the
   find operation and explain why.
 
-An external package with ``<Package>_IS_FULLY_TRIBITS_COMPLIANT=TRUE`` **AND**
+An external package with ``<Package>_IS_TRIBITS_COMPLIANT=TRUE`` **AND**
 ``<Package>_PROCESSED_BY_DOWNSTREAM_TRIBITS_EXTERNAL_PACKAGE=FALSE`` is the
 one for which ``find_package(<Package> CONFIG REQUIRED)`` will be called and
 does not have any downstream packages that are being treated as external
 packages.
 
-The variable ``<Package>_IS_FULLY_TRIBITS_COMPLIANT`` is set right when the
+The variable ``<Package>_IS_TRIBITS_COMPLIANT`` is set right when the
 packages are initially defined by reading in the various input files.  That
 is, all initially internal packages that are listed in a
 `<repoDir>/PackagesList.cmake`_ file will have
-``<Package>_IS_FULLY_TRIBITS_COMPLIANT=TRUE`` set.  While all external
+``<Package>_IS_TRIBITS_COMPLIANT=TRUE`` set.  While all external
 packages/TPLs listed in a `<repoDir>/TPLsList.cmake`_ file will have
-``<Package>_IS_FULLY_TRIBITS_COMPLIANT=FALSE`` set (except for those tagged
+``<Package>_IS_TRIBITS_COMPLIANT=FALSE`` set (except for those tagged
 with ``TRIBITS_PKG`` which will have
-``<Package>_IS_FULLY_TRIBITS_COMPLIANT=FALSE`` set).
+``<Package>_IS_TRIBITS_COMPLIANT=FALSE`` set).
 
-NOTE: When a TriBITS TPL (i.e. ``<Package>_IS_FULLY_TRIBITS_COMPLIANT=FALSE``)
+NOTE: When a TriBITS TPL (i.e. ``<Package>_IS_TRIBITS_COMPLIANT=FALSE``)
 is being processed, we can't assume where its
 ``<UpstreamPackage>Config.cmake`` file exists so we must find upstream
 dependencies using ``set(<UpstreamPackage>_DIR ...)`` and
 ``find_dependency(<UpstreamPackage> CONFIG REQUIRED)``.
 
 So the first loop over external packages/TPLs will be those external
-packages/TPLs that have ``<Package>_IS_FULLY_TRIBITS_COMPLIANT=TRUE`` **OR**
+packages/TPLs that have ``<Package>_IS_TRIBITS_COMPLIANT=TRUE`` **OR**
 ``<Package>_PROCESSED_BY_DOWNSTREAM_TRIBITS_EXTERNAL_PACKAGE=TRUE``.  And we
-only call ``find_package()`` for those fully TriBITS-complient external
```

Comment:
That sounds like it is worth looking into.  Would you have the time and interest to look into adding this?  It would likely be best to create a TriBITS ctest test that runs codespell on the code base and then reports errors locally and on CDash.

Reply:
I added the issue #571 to look into adding automated spell checking.

@KyleFromKitware, would you be interested in helping to set that up?

Reply:
Sure, I could work on that.